app/models/assignment_journal.rb

```diff
@@ -0,0 +1,5 @@
+class AssignmentJournal < ApplicationRecord
+  belongs_to :volunteer
+  belongs_to :assignment
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
I think the `foreign_key` is not needed here, it will already use the association name to build it.